fix and enhance extract of /etc/salt/master.d/susemanager.conf (#206)
uyuni-project/uyuni-docs: modules/salt/pages/salt-file-locations.adoc

= Salt File Locations and Structure

There are several ways to set up the Salt file structure.

> Here it is described how Salt is supported and set up as part of {productname} Server.

Reviewer: ```suggestion
This section describes how Salt is supported and set up as part of {productname} Server.
```

The main configuration file is [path]``/etc/salt/master.d/susemanager.conf``.

[NOTE]
====

> To avoid trouble in the future better do not touch [path]``/etc/salt/master.d/susemanager.conf``.

Reviewer: ```suggestion
Do not edit the [path]``/etc/salt/master.d/susemanager.conf`` configuration file.
```

This file belongs to the [package]``spacewalk-setup`` package and is marked as [literal]``%config``.

> When {suse} updates [package]``spacewalk-setup`` [path]``susemanager.conf`` can get overwritten and customization can get lost.

Reviewer: ```suggestion
When {suse} updates the [package]``spacewalk-setup`` package, the [path]``susemanager.conf`` file is overwritten, and any customization is lost.
```

> Whenever possible it is recommended to add own configuration file to the [path]``/etc/salt/master.d/`` directory, and thus preventing the update process from deleting own settings from the main [path]``susemanager.conf`` configuration file.

Reviewer: ```suggestion
Instead, add your own configuration file to the [path]``/etc/salt/master.d/`` directory.
This prevents the update process from deleting your settings from the main [path]``susemanager.conf`` configuration file.
```

====

> Here {suse} lists some settings from [path]``/etc/salt/master.d/susemanager.conf`` that will help with finding configuration options:

Reviewer: ```suggestion
Some settings from [path]``/etc/salt/master.d/susemanager.conf`` that can help with finding configuration options:
```

[path]``/usr/share/salt-formulas/states``::
[path]``/usr/share/susemanager/formulas/states``::
These directories are shipped and updated with {productname} or additional extensions.

> They inclucde states for salt formulas.

Reviewer: ```suggestion
They include states for Salt formulas.
```
